src/devices/Hx711I2c/Hx711I2c.cs

```diff
@@ -0,0 +1,394 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Buffers.Binary;
+using System.Device.Gpio;
+using System.Device.I2c;
+using System.Device.Model;
+using System.Diagnostics;
+using System.Threading;
+using UnitsNet;
+
+namespace Iot.Device.Hx711
```

Comment:
What about setting the namespace declaration on a single line? (Terminating it with ';')